.github/CONTRIBUTING.md

```diff
@@ -175,64 +213,56 @@ There are checks that automation will not be able to capture. For example:
    -  Windows 10
       -  Chrome
       -  Microsoft Edge Chromium
-      -  Microsoft Edge UWP (Legacy)
       -  Firefox
-      -  IE11 (except speech features)
+      -  Internet Explorer 11 (except speech features)
    -  Safari on macOS
    -  Safari on iOS or iPadOS
    -  Chrome on Android
--  For accessibility, please refer to [`docs/ACCESSIBILITY.md`](https://github.com/microsoft/BotFramework-WebChat/blob/main/docs/ACCESSIBILITY.md)
-   -  Tab order, content readability, assistive technology-only text, color contrast, etc. must be maintained.
-   -  Assistive technology and browser compatibility
-      -  NVDA/JAWS: Chrome and Firefox
-      -  Narrator: Microsoft Edge family and IE11
-      -  VoiceOver: Safari
-      -  TalkBack: Chrome on Android
--  For internationalization, please refer to [`docs/LOCALIZATION.md`](https://github.com/microsoft/BotFramework-WebChat/tree/main/docs/LOCALIZATION.md)
 -  Feature documentation, samples, live demo, operation of demo bots
    -  All samples must also come with a hosted live demo
    -  Please discuss with us if a specific bot is needed for the live demo
 
-# Workflow
+## Our workflows
 
 Here list how we generally work when [fixing a bug](#fixing-bug) or [implementing a new feature](#implement-new-feature).
 
-## Fixing bug
+### Fixing bugs
 
 Write the bug repro as a test, before fixing the bug.
 
-1. Clone/prepare the repository or reset an existing repository
-   -  To reset the repo, run `npm run tableflip`
-1. Run continuous build, run `npm start`
-1. Convert bug into a test case, under `__tests__/html/this-is-the-bug.html`
-1. Run the test, `npm run start:docker`, then `npm test -- --testPathPattern html/this-is-the-bug.html`
+1. Clone and prepare the repository or reset an existing one
+   -  To reset, run `npm run tableflip`
+1. Run continuous build by running `npm start`
+1. Reproduce the bug on a test page, under `__tests__/html/this-is-the-bug.html`
```

Comment:
```suggestion
1. Reproduce the bug on a test page under `__tests__/html/this-is-the-bug.html`
```
Add note about following test naming conventions

Reply:
isn't the naming convention actually `this.is.the.bug.html`?

Reply:
Naming the test file is difficult. 😅

Let me rename it to `this.is.the.bug.html`.